ruby/vampires.rb

```diff
@@ -0,0 +1,95 @@
+puts "What is your name?"
+user_name = gets.chomp.to_s
+
+puts "How old are you?"
+age = gets.chomp.to_i 
+
+puts "What year were you born?"
+year = gets.chomp.to_i
+
+current_year = 2017
```

Comment:
Check out the Ruby Time class and see what you can do with `Time.now.year`.  This will help the code run a but more dynamically.  Otherwise you will have to keep changing this file every new year to keep the code running properly.